src/generator.js

```diff
@@ -33,11 +41,35 @@ function createPHPFile(folder) {
             return
         }
 
+        const category = detectCategory(name);
+
+        if (category == 'class') {
+            return window.showQuickPick([
+                "Class",
+                "Trait",
+                "Interface"
+            ], {
+                title: "Choose class type",
+                canPickMany: false,
+                matchOnDescription: true,
+                matchOnDetail: true
+            });
+        }
```

Comment:
Why bothering users with another question, most of the time they'll choose "class" ?

Let them change manually the word `class` by whatever they want

```suggestion
```

Reply:
I think the target is create a class that do not need to modify, and just add our logic code.
Actully when detect is trait or interface, quick pick will not show.

In PHPStorm when create a class, we can choose what template (type) to create class, and we can choose what class to extended, and what interface to implements. This is very helpful.

![WX20230627-103922](https://github.com/blanc-frederic/vs-phpcompanion/assets/8629617/f1d5cf03-a723-49a9-81a2-0ca36b8fa6a6)